### PR TITLE
MAINT: around label and regionprops functions.

### LINKED ISSUE #2116: not clearly description of label bbox in documentation
Hello.
So, I use bboxes of founded labels to create their copies and put them side by side with original ones. And it wasn't obviosly for me (as for newbie) that bbox's (max_row, max_col) is actually (max_row + 1, max_col + 1), as I found out in sources.

(How I thought | How it is)
image

So I think it would be nice to clarify this in the docs.

Also, maybe you should add something like bbox_area in region properties. I know about label.image.size, but it not obvious to.
Thanks.

### PR DESCRIPTION
**Pull request by sciunto**
CHANGELOG

* [x] move docstring from pyx to py
* [x] move references from header to docstring
* [x] merge two cython functions (label, a wrapper of _label, and _label)
* [x] add cross ref between these functions (label and regionprops)
* [x] PEP8
* [ ] fix #2116

### CODE REVIEW

File: skimage/measure/_ccomp.pyx [hunk 1]
```diff
@@ -357,104 +342,14 @@ def undo_reshape_array(arr, swaps):
     return reshaped
 
 
-# Connected components search as described in Fiorio et al.
-def label(input, neighbors=None, background=0, return_num=False,
+
+def label(input, neighbors=None, background=None, return_num=False,
```
Comment (soupault) [inline skimage/measure/_ccomp.pyx]: Please, rename this to `_label_cython` and change the corresponding import in `.py`.

**Comment by soupault** [comment]
LGTM.

**Comment by sciunto** [comment]
Done. Let's wait for travis ;)

**Comment by sciunto** [comment]
@soupault Travis fails and I can't reproduce. Any idea?

**Comment by sciunto** [comment]
OK, i fixed one problem, there is still another one with the references. A warning treated as an error, WARNING: duplicate citation R303 (which corresponds to the first one).

**Comment by sciunto** [comment]
One test failed because of #2113

**Comment by soupault** [comment]
:+1:

**Comment by jni** [comment]
Thanks @sciunto! I'll leave `bbox_area` for another PR, since it's somewhat independent.

**Comment by sciunto** [comment]
Yes :+1:

**Pull request merged**